### PR TITLE
Update Desktop Modeler connector template documentation

### PR DESCRIPTION
**Pull request by nikku**
## Description

As per https://github.com/camunda/camunda-modeler/issues/4455 pre-built connector templates are now offered automatically within the Desktop Modeler. This PR reflects that change in documentation and flags.

This also backports the changes to 8.5, the latest released version.

Related to https://github.com/camunda/camunda-modeler/pull/4474

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [x] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [ ] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [x] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [x] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

----

This is a follow-up for https://github.com/camunda/camunda-docs/pull/3870.

### CODE REVIEW

**Review by barmac — DISMISSED** [review]

**Comment by nikku** [comment]
https://github.com/camunda/camunda-docs/pull/4187/commits/0c69b27f66d65b03556d308ecc098f72775376f0 added to support https://github.com/camunda/camunda-modeler/pull/4479.

File: docs/components/modeler/desktop-modeler/flags/flags.md [hunk 1]
```diff
@@ -153,14 +153,14 @@ To use the new context pad, configure your `flags.json` as follows:
...
 
-To enable the automatic fetching of Camunda 8 Connector templates, configure your `flags.json` as follows:
+<span class="badge badge--cloud">Camunda 8 only</span>
+
+To [disable automatic Connector template fetching](../use-connectors.md#automatic-connector-template-fetching), configure your `flags.json` as follows:
 
 ```json
 {
-  "enable-connector-templates": true
+  "enable-connector-templates": false
```
Comment (barmac) [inline docs/components/modeler/desktop-modeler/flags/flags.md:164]:
```suggestion
  "disable-connector-templates": true
```

File: docs/components/modeler/desktop-modeler/use-connectors.md [hunk 2]
```diff
@@ -24,11 +27,15 @@ Once fetched, you can use the templates in the Camunda 8 BPMN editor.
...
 
-For greater control, you can download individual Camunda 8 Connector templates manually through the [Camunda Marketplace website](https://marketplace.camunda.com/) and make them available in Desktop Modeler via the [standard element template search paths](./element-templates/configuring-templates.md).
+For full control over building blocks offered by the Desktop Modeler, download templates for individual Camunda 8 Connectors manually, i.e. through the [Camunda Marketplace](https://marketplace.camunda.com/). Make them available to the Desktop Modeler via the [standard element template search paths](./element-templates/configuring-templates.md).
 
 ## Additional resources
 
-- [Learn about Camunda 8 Connectors](../../connectors/out-of-the-box-connectors/available-connectors-overview.md)
-- [Desktop Modeler flags](./flags/flags.md)
+- [About Camunda 8 Connectors](../../connectors/out-of-the-box-connectors/available-connectors-overview.md)
+- [Desktop Modeler flags](./flags/flags.md#enable-connector-templates)
```
Comment (barmac) [inline docs/components/modeler/desktop-modeler/use-connectors.md:41]:
```suggestion
- [Desktop Modeler flags](./flags/flags.md#disable-connector-templates)
```

File: versioned_docs/version-8.5/components/modeler/desktop-modeler/use-connectors.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+Disable automatic fetching of Connector templates through the [respective flag](./flags/flags.md#disable-connector-templates).
+
+## Add Connector templates manually
+
+For full control over building blocks offered by the Desktop Modeler, download templates for individual Camunda 8 Connectors manually, i.e. through the [Camunda Marketplace](https://marketplace.camunda.com/). Make them available to the Desktop Modeler via the [standard element template search paths](./element-templates/configuring-templates.md).
+
+## Additional resources
+
+- [About Camunda 8 Connectors](../../connectors/out-of-the-box-connectors/available-connectors-overview.md)
+- [Desktop Modeler flags](./flags/flags.md#enable-connector-templates)
```
Comment (barmac) [inline versioned_docs/version-8.5/components/modeler/desktop-modeler/use-connectors.md:41]:
```suggestion
- [Desktop Modeler flags](./flags/flags.md#disable-connector-templates)
```

**Comment by barmac** [comment]
There are still some places to update with the flag change, but overall it looks good.

**Comment by nikku** [comment]
My appologies. Not sure how I missed these changes :see_no_evil:

**Review by barmac — APPROVED** [review]

**Review by conceptualshark — CHANGES_REQUESTED** [review]
I pushed a few changes to clean up the organization/linking on this page, and this seems pretty good to go. 

However, the old `enable` version of the feature flag is still referenced in the [release notes](https://docs.camunda.io/docs/next/reference/release-notes/860/#using-camunda-8-connectors-in-desktop-modeler). I couldn't find any updated copy for this section, and wasn't sure if it should come from Product vs my own edits.

**Comment by nikku** [comment]
@conceptualshark Cf. https://github.com/camunda/camunda-docs/pull/4187/commits/27cabcdee0e62fbe5cfa1b2246e0c692e8ca9ab4#r1732955161. I'll revert the change of URL and update the release notes according to https://github.com/camunda/camunda-docs/pull/4187#issuecomment-2312705364.

**Review by nikku — COMMENTED** [review]
@conceptualshark Please see my latest updates

* restoring the link
* re-phrasing the 8.6 release notes section

**Review by conceptualshark — DISMISSED** [review]
Thanks for the catch on the link, and the updates! LGTM! 👍

File: docs/reference/release-notes/860.md [hunk 4]
```diff
@@ -27,10 +27,12 @@ keywords:
 
 <!-- https://github.com/camunda/product-hub/issues/2253 -->
 
-You can now enable automatic fetching of Camunda 8 Connector templates in Desktop Modeler, so you can always use the latest templates in your BPMN diagrams. This feature enhances workflow efficiency and reduces setup time.
+The Modeler now makes templates for pre-built Camunda 8 connectors available for use in BPMN diagrams. This ensures you can easily connect to a growing list of services and protocols, out of the box.
```
[the remark this answers is not in the document]
Comment (christinaausley) [reply docs/reference/release-notes/860.md:30]: Regardless of wording, we do need to uppercase Connectors consistently 😄

Comment (nikku) [inline docs/reference/release-notes/860.md:30]:
```suggestion
Modeler now offers pre-built templates in BPMN diagrams for Camunda 8 Connectors. This ensures you can easily connect to a growing list of services and protocols, out of the box.
```

[the remark this answers is not in the document]
Comment (nikku) [reply docs/reference/release-notes/860.md]: Applied, thanks.

**Review by christinaausley — APPROVED** [review]

**Pull request merged**